### DISCUSSION TITLE
R2 Mod Manager - Game Mode

### DISCUSSION THREAD

**Discussion by SilvertongueRED** [ebkr/r2modmanPlus#1113] [Q&A]
Anyone know how to get R2 Mod Manager working in game mode? Adding as non steam game doesn’t work. I have the app image working fine in desktop mode. Would make a world of difference when playing with mods with friends to just do it all from game mode!

**Comment by MaddHatterHX** [comment]
We need to generate some traction on this issue.

**Reply by SilvertongueRED** [reply]
Hi! Sorry I never replied to this originally. So, I figured it out: 
I used this free application to convert the R2 Mod Manager AppImage to a “addable” and workable non-steam game. R2 Mod Manager works PERFECTLY in game mode now! 
https://github.com/TheAssassin/AppImageLauncher 

Hope this helps lemme know if you have any questions with it! It’s been awhile since I had to use it now but I’ll do my best to assist!

**Reply by SilvertongueRED** [reply]
Found the Reddit post explaining better!! Just installed R2 this way on another friend’s deck works great from steam even in game mode! It’s this Lite version you want actually. 
https://www.reddit.com/r/SteamDeck/comments/t9xwte/how_to_automatically_integrate_appimage_apps_into/
